solutions/devsprint-cecilia-saraiva-2/FinanceAppSnapshotTests/ActivityViewCellSnapshotTests.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  ActivityViewCellSnapshotTests.swift
+//  FinanceAppTests
+//
+//  Created by Raline Maria da Silva on 24/08/22.
+//
+
+import XCTest
+import SnapshotTesting
+@testable import FinanceApp
+
+class AcitivityCellViewSnapshotTests: XCTestCase {
+    
+    let size1: CGSize = .init(width: 320, height: 120)
+    let size2: CGSize = .init(width: 420, height: 120)
```

Comment:
[sugestao] que tal usarmos nomes mais significativos? Alem disso, como boa pratica, podemos colocar `private` tambem.
```suggestion
    private let size320x120: CGSize = .init(width: 320, height: 120)
    private let size420x120: CGSize = .init(width: 420, height: 120)
```

Reply:
feito